### ISSUE TITLE
folders: delete api return no json since 9.3

### ISSUE THREAD

**Issue by rrey** [grafana/grafana#77673]
### What happened?

The folders api endpoint do not return json in its response on a DELETE operation.

The documentation shows an example where the response shall return json ([doc](https://grafana.com/docs/grafana/v9.5/developers/http_api/folder/#delete-folder)) but when testing there isn't any json returned.

I detected the issue on 9.5.13 but It seems to be introduced in 9.3.0 since 9.2.20 was still returning json.

### What did you expect to happen?

json response on delete folder.

### Did this work before?

Yes, last version returning json was 9.2.20

### How do we reproduce it?

1. create a folder
2. delete it through the API endpoint (`curl -u "admin:admin" -X DELETE http://127.0.0.1:3000/api/folders/<folder-uid>`)
3. Check the response.

Example on 9.2.20 and earlier:
```
curl -u "admin:admin" -X DELETE http://127.0.0.1:3000/api/folders/N4O-a04Iz
{"id":2,"message":"Folder toto deleted","title":"toto"}%
```
Now the response is empty

### Environment (with versions)?

Grafana: 9.3.0 and later
OS: container
Browser: N/A

### Grafana platform?

Docker

**Comment by tonypowa** [comment]
hi @rrey 
thanks for reporting this issue
I reproduced the empty message with status code 200 OK when deleting a folder via `/api/folders/` endpoint
Forwarding issue to back-end platform squad

**Comment by mildwonkey** [comment]
HI @rrey , thanks for reporting this and sorry for the unexpected behavior! 

This issue has been fixed in Grafana 10.0+ (and I've confirmed this), so I am going to close this. When you upgrade to the newer version you will see the expected response.

**Comment by rrey** [comment]
Thanks !
